src/domain/__tests__/data.ts

```diff
@@ -192,3 +191,8 @@ export const consumeEventStreamRecordDelivered = {
     })),
   },
 };
+
+export const acceptedNotification = makeFullSentNotification(aSenderPaId)(aDate)({
+  ...newNotificationRequest,
+  notificationRequestId: notificationId.valid,
+})(aIun.valid);
```

Comment:
[Optional]: I suggest the following change to improve the consistency with other tests, in addition in the future we may require `GetNotificationDetailRecord` data :)

```suggestion
// GetNotificationDetailRecord /////////////////////////////////////////////////

const acceptedNotification = makeFullSentNotification(aSenderPaId)(aDate)({
  ...newNotificationRequest,
  notificationRequestId: notificationId.valid,
})(aIun.valid);

export const getNotificationDetailRecordAccepted: GetNotificationDetailRecord = {
  type: 'GetNotificationDetailRecord',
  input: { apiKey: apiKey.valid, iun: aIun.valid },
  output: { statusCode: 200, returned: acceptedNotification }
};
```

Reply:
Done!